.github/ISSUE_TEMPLATE/hotfix-template.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Notify Hotfix Template
+about: Create hotfix ticket for urgent PRs
+title: 'HOTFIX: '
+labels: Notify, hotfix
+assignees: ''
+
+---
+
+## Description - What Problem Does This Solve?
+
+<!--_**Required.** Briefly describe the problem below this line. Note that all of this can be filled in when the PR is in review_-->
+
+- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).
+- [ ] Sprint is added to this ticket and ticket is moved to In Progress
+- [ ] Point value is assigned to this ticket when the effort is determined
+- [ ] This work is added to the sprint review slide deck (key win bullet point and demo slide)
```

Comment:
```suggestion
<!--_**Required.** Briefly describe the problem below this line. Note that all of this can be filled in when the PR is in review_-->

## Expected Outcomes
- [ ] Ticket is understood, and QA has been contacted (if the ticket has a QA label).
- [ ] Sprint is added to this ticket and ticket is moved to In Progress
- [ ] Point value is assigned to this ticket when the effort is determined
- [ ] This work is added to the sprint review slide deck (key win bullet point and demo slide)
```

Reply:
There was an extra line under the description. Figured the checkboxes should have their own section.